controllers/users.js

```diff
@@ -22,37 +22,42 @@ exports.users_post_register = async (req, res) => {
 
   // create user
   const newUser = {
-    id: uuid.v4(),
+    id: uuidv4(),
     name: req.body.name,
     email: req.body.email.toLowerCase(),
     password: req.body.password,
   };
 
   try {
     await users.push(newUser);
-    res.send(newUser);
+    res.json({
+      id: newUser.id,
+      name: newUser.name,
+      email: newUser.email,
```

Comment:
How about you destructure this
```suggestion
     const { id, name, email } = newUser;
      id,
      name,
      email,
```